app/models/recipe.rb

```diff
@@ -0,0 +1,10 @@
+class Recipe < ApplicationRecord
+  has_many :ingredients, dependent: :destroy
+  has_many :foods, through: :ingredients
+  belongs_to :user
+
+  # validates :name, presence: true
+  # validates :preparation_time, presence: true
+  # validates :cooking_time, presence: true
+  # validates :description, presence: true
```

Comment:
### [OPTIONAL]
It's a good practice to validate a model but in case if you don't want to validate the model then kindly remove the commented code to make the file look clean and professional. 👍